src/components/ui/Rotation.tsx

```diff
@@ -37,17 +38,20 @@ export default class Rotation extends Component<RotationProps> {
 					return false
 				}
 
-				const linkClassName = [styles.link]
-				if (!action.onGcd) {
-					linkClassName.push(styles.ogcd)
-				}
+				const linkClassName = [
+					styles.link,
+					{[styles.ogcd]: !action.onGcd},
+				]
+
+				const iconSize = action.onGcd ? styles.gcdSize : styles.ogcdSize
```

Comment:
You may have forgotten about `@value` but i sure as fuck had forgotten about using it in js from the css import.

Reply:
:yousmart: